src/source/Ice/IceAgent.c

```diff
@@ -36,6 +36,7 @@ STATUS createIceAgent(PCHAR username, PCHAR password, PIceAgentCallbacks pIceAge
     pIceAgent->tieBreaker = (UINT64) RAND();
     pIceAgent->iceTransportPolicy = pRtcConfiguration->iceTransportPolicy;
     pIceAgent->kvsRtcConfiguration = pRtcConfiguration->kvsRtcConfiguration;
+    NULLABLE_SET_EMPTY(pIceAgent->rtcIceMetrics.rtcIceCandidatePairStats.circuitBreakerTriggerCount);
```

Comment:
I know these were in the branch. Not sure if you rebased off top of the branch. Let us remove this. Same with all ICE candidate related stats. Thanks.